quickstart/101-managed-instance/main.tf

```diff
@@ -0,0 +1,85 @@
+# Create resource group
+resource "azurerm_resource_group" "example" {
+  name     = var.azurerm_resource_group_name
+  location = var.location
+}
+
+# Create security group
+resource "azurerm_network_security_group" "example" {
+  name                = var.azurerm_network_security_group_name
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+}
+
+# Create a virtual network
+resource "azurerm_virtual_network" "example" {
+  name                = var.azurerm_virtual_network_name
+  resource_group_name = azurerm_resource_group.example.name
+  address_space       = ["10.0.0.0/16"]
```

Comment:
Why such a big network for this example? Can you use /24 instead?

Reply:
yes this was random

Reply:
@urosran Please update the Vnet and Subnet examples with smaller networks.